### ISSUE TITLE
the in-progress icon float layer out

### ISSUE THREAD

**Issue by marslo** [afonsof/jenkins-material-theme#183]
Jenkins version: 2.284/2.283
simple theme plugin version: 0.6
material theme: 0.3.2

The running icon shows up:
<img src="https://user-images.githubusercontent.com/5822057/112141523-577b5d00-8c10-11eb-961d-6fc41ce5075d.gif" width="30%">

after using Jenkins default theme:
<img src="https://user-images.githubusercontent.com/5822057/112141582-66620f80-8c10-11eb-91ec-54a28a44bdcc.gif" width="30%">

**Comment by starwarsfan** [comment]
Second that, same here.

**Comment by Smasherr** [comment]
@starwarsfan I thought it was a feature 🤣

**Comment by mit4ever** [comment]
Metoo, maybe problem is jenkins

**Comment by marslo** [comment]
> Metoo, maybe problem is jenkins

Jenkins default theme works well with the "in progress" status icon. So that quite unlikely is the issue of Jenkins.

**Comment by mit4ever** [comment]
> > Metoo, có thể vấn đề là jenkins
> 
> Chủ đề mặc định của Jenkins hoạt động tốt với biểu tượng đang chạy. Vì vậy, điều đó khá khó xảy ra là vấn đề của Jenkins.

I agree. Used  to be fine but when I recently updated jenkins there was a problem with icon

**Comment by marslo** [comment]
well... I've using workaround bypass the issue :
```
svg[class*=anime] { visibility: collapse }
```

btw, using `hidden` as the same as the `collapse`.

Details:
<img src="https://user-images.githubusercontent.com/5822057/112464977-21201800-8d9f-11eb-9793-c0464f888196.gif" width="100%">


it can be setup in Jenkins via:
<img width="1021" alt="simple-theme-7" src="https://user-images.githubusercontent.com/5822057/112458835-334a8800-8d98-11eb-8f47-6aa05d44d33a.png">

**Comment by Smasherr** [comment]
Nice job @marslo! Would be cool to see a PR for this

**Comment by marslo** [comment]
> Nice job @marslo! Would be cool to see a PR for this

this is actually the workaround, just hidden the overlayer svg image...

**Comment by fguillotpro** [comment]
Still happening with Jenkins 2.293 and jenkins-material-theme.css v.1.3.3, any news on that matter? Thanks

**Comment by eplodn** [comment]
Implemented the workaround (thanks @marslo !) but it would be great if @afonsof or somebody could fix the underlying issue.

**Comment by offa** [comment]
Related Jenkins Issue: [**JENKINS-65808**](https://issues.jenkins.io/browse/JENKINS-65808)

**Comment by offa** [comment]
The issue has hit other themes too: https://github.com/TobiX/jenkins-neo2-theme/issues/40

**Comment by pboiseau** [comment]
Hello, thank you for the workaround. 

If anyone want to set up it using groovy script, here is the script I use 

```groovy
import jenkins.model.Jenkins
import org.jenkinsci.plugins.simpletheme.CssUrlThemeElement
import org.jenkinsci.plugins.simpletheme.CssTextThemeElement

Jenkins jenkins = Jenkins.get()

def themeDecorator = jenkins.getExtensionList(org.codefirst.SimpleThemeDecorator.class).first()

themeDecorator.setElements([
        new CssUrlThemeElement('https://cdn.rawgit.com/afonsof/jenkins-material-theme/gh-pages/dist/material-indigo.css'),
        // fix jenkins theme issue https://github.com/afonsof/jenkins-material-theme/issues/183#issuecomment-806518351
        new CssTextThemeElement('svg[class*=anime] { visibility: collapse }')
])

jenkins.save()
```

**Comment by abstract-entity** [comment]
> well... I've using workaround bypass the issue :
> 
> ```
> svg[class*=anime] { visibility: collapse }
> ```
> 
> btw, using `hidden` as the same as the `collapse`.
> 
> Details: <img alt="" width="100%" src="https://user-images.githubusercontent.com/5822057/112464977-21201800-8d9f-11eb-9793-c0464f888196.gif">
> 
> it can be setup in Jenkins via: <img alt="simple-theme-7" width="1021" src="https://user-images.githubusercontent.com/5822057/112458835-334a8800-8d98-11eb-8f47-6aa05d44d33a.png">

Thanks, it do the job

**Comment by jiuhuche120** [comment]
ths
